prestodb/client.py

```diff
@@ -37,9 +37,11 @@
 import os
 from typing import Any, Dict, List, Optional, Text, Tuple, Union  # NOQA for mypy types
 
-import prestodb.logging
```

Comment:
Why are we removing prestodb.logging here? We need it on line 52.

Reply:
Oops, my bad. It got optimized away in pycharm..